resources/js/api/index.ts

```diff
@@ -40,6 +62,12 @@ export class ApiService {
             mode,
         });
 
+        if (resp.status === 419 && nest && useAppStore().isMultiTenant) {
```

Comment:
<img width="1440" alt="Screenshot 2024-01-15 at 11 31 51 AM" src="https://github.com/enjin/platform-ui/assets/114082473/314eb6d5-cbb8-40f3-bff1-54a40d255788">

This causes infinite loop, I think you need a refresh token endpoint for this

Reply:
Seem to be an issue on my local